cmake/catkinConfig.cmake.in

```diff
@@ -10,6 +10,37 @@
 # :outvar <comp>_INCLUDE_DIRS/_LIBRARY_DIRS/_LIBRARY:
 #    contains the include dirs / library dirs / libraries of the searched component <comp>.
 
+if(APPLE)
+  # Turn absolute RPATH directories on to avoid issues when DYLD_LIBRARY_PATH
+  # is not set. In 10.11 (el capitan) the SIP security feature prevents passing
+  # these DYLD_* environment variables down to sub-shells.
+  macro(_set_cmake_policy_to_new_if_available policy)
+    if(POLICY ${policy})
+      cmake_policy(SET ${policy} NEW)
+    endif()
+  endmacro()
```

Comment:
This macro is already being defined in the `all.cmake` file which is being included a few lines below.